src/component/person.jsx

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable max-len */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './person.scss';
+
+const Married = (isMarried, sex, partnerName) => {
+  let message = '';
+
+  if (isMarried === false) {
+    message = 'I am not married';
+  } else if (isMarried === true) {
+    if (sex === 'f') {
+      message = `My husband's name is ${partnerName}`;
+    } else {
+      message = `My wife's name is ${partnerName}`;
+    }
+  }
+
+  return message;
+};
+
+export const Person = ({ person }) => {
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+      <p className="Person__age">{person.age ? `I am ${person.age}` : ''}</p>
+      <p className="Person__partner">{Married(person.isMarried, person.sex, person.partnerName)}</p>
```

Comment:
Will be good to create separate variable before rendering section, save here the result of this function and use this variable in jsx